examples/activities/resample.js

```diff
@@ -23,21 +23,29 @@ module.exports = function resample(localInputPath, outputUri, options, callback)
       return callback(err);
     }
 
+    // TODO: Shouldn't assume target SRS.
     var args = [
       "-q",
-      "-t_srs", "EPSG:3857",
+//      "-t_srs", "EPSG:3857",
       "-te", options.targetExtent[0], options.targetExtent[1], options.targetExtent[2], options.targetExtent[3],
-      "-tr", options.targetResolution[0], options.targetResolution[1],
-      "-tap",
+//      "-tr", options.targetResolution[0], options.targetResolution[1],
+//      "-tap",
+      "-ts", "256", "256",
       "-wm", 256, // allow GDAL to work with larger chunks (diminishing returns after 500MB, supposedly)
       "-wo", "NUM_THREADS=ALL_CPUS",
       "-multi",
       "-co", "tiled=yes",
-      "-co", "compress=lzw",
-      "-co", "predictor=2",
       "-r", "bilinear"
     ];
 
+    if(!options.nocompression) {
```

Comment:
I let no compression be an option, so that if this is an intermediate step in the local case we don't pay compression CPU costs. This might be useful for when we can compose activities.